force-app/main/default/lwc/changeAmountOrFrequencyModal/changeAmountOrFrequencyModal.html

```diff
@@ -0,0 +1,25 @@
+<template>
+  <template if:true={open}>
+    <section role="dialog" tabindex="-1" aria-modal="true" aria-labelledby="modal-heading-01" class="slds-modal slds-fade-in-open">
```

Comment:
If slds-modal is used to build the custom modal, Then javascript focus trapping should be implement to ensure that the user can only focus on the modal.

Reply:
good suggestion! focus trapping added